CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Package instructions
 - Add instructions in README where people can find signed extension (see [issue 22](https://github.com/KNowledgeOnWebScale/solid-authentication-browser-extension/issues/22))
 - Mention the minimum version of CSS that is needed (see [issue 23](https://github.com/KNowledgeOnWebScale/solid-authentication-browser-extension/issues/23))
+- Domain filter (with optional regex syntax)
```

Comment:
Mention issue https://github.com/KNowledgeOnWebScale/solid-authentication-browser-extension/issues/24

Reply:
Added in c55cf1c404d5cb89e58bc683f24315792364309f